cmd/cmd.go

```diff
@@ -614,11 +617,53 @@ func versionString(manager *Manager) string {
 	if GitHash != "" {
 		suffix = fmt.Sprintf(" hash %s\n", GitHash)
 	}
-	return fmt.Sprintf("%s version %s.%s", manager.name, manager.version, suffix)
+	return fmt.Sprintf("Client version: %s.%s", manager.version, suffix)
+}
+
+func apiVersionString(client *Client) (string, error) {
+	if client == nil {
+		return "", fmt.Errorf("client cannot be nil")
+	}
+
+	url, err := GetURL("/info")
+	if err != nil {
+		return "", err
+	}
+
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", err
+	}
+
+	resp, err := client.Do(req)
+	if err != nil {
+		// if we return the error, stdout won't flush until the prompt
+		return fmt.Sprintf("Unable to retrieve server version: %v", err), nil
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	defer resp.Body.Close()
```

Comment:
Only makes sense close the Body reader when no error is returned, move this statement to below of error checking.